packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -248,13 +241,11 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
                 </StyledCheckbox>
             </StyledCard>
 
-            <StyledTooltipButton
-                onClick={anonymize}
-                isDisabled={isDisabled}
-                tooltipContent={getButtonTooltipMessage()}
-            >
-                <Translation id="TR_ANONYMIZE" />
-            </StyledTooltipButton>
+            <Tooltip content={getButtonTooltipMessage()}>
+                <StyledButton onClick={anonymize} isDisabled={isDisabled}>
+                    <Translation id="TR_ANONYMIZE" />
```

Comment:
By the design, there is a new copy different from the one in BalanceSection ("Start coinjoin" vs "Anonymize") so maybe it makes sense to create a new message id here.

Reply:
5eca8e7fc665902d1c5124883fa1cefb2034f08f